tests/components/vesync/test_coordinator.py

```diff
@@ -0,0 +1,92 @@
+"""Tests for the coordinator."""
+
+from datetime import timedelta
+
+from freezegun.api import FrozenDateTimeFactory
+import requests_mock
+
+from homeassistant.components.vesync.const import DOMAIN, UPDATE_INTERVAL
+from homeassistant.config_entries import ConfigEntryState
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME, STATE_UNAVAILABLE
+from homeassistant.core import HomeAssistant
+
+from .common import (
+    mock_air_purifier_400s_update_response,
+    mock_device_response,
+    mock_multiple_device_responses,
+    mock_outlet_energy_response,
+)
+
+from tests.common import MockConfigEntry, async_fire_time_changed
+
+
+async def test_coordinator_update(
```

Comment:
This is rather testing behavior of the entities, so let's move this to a platform test file instead

Reply:
Renamed the file and test